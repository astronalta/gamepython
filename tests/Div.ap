# aptest exec on


main() Int {

    Io::println((16/2).str)
    Io::println((7/3).str)

    Io::println((7*6).str)

    ret 0
}
