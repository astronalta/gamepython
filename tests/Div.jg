# jgtest exec on

Obj < Object {
    x = 99

    test() Int { ret x / 33 }
    
}


main() Int {

    Io::println((16/2).str)
    Io::println((7/3).str)

    Io::println((7*6).str)
    Io::println(Obj().test().str)

    ret 0
}
