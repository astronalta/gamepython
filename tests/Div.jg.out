8
2
42
3
